compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_loops_for_vectorization.mlir

```diff
@@ -246,4 +227,30 @@ module {
   }
 }
 
+// -----
 
+// CHECK-LABEL: @element_wise
+module {
+  func.func @element_wise(%arg0: tensor<1x1x4x4x4x4xf32>, %arg1: tensor<1x1x4x4x4x4xbf16>) -> tensor<1x1x4x4x4x4xbf16>{
```

Comment:
Can you go over this one more time an simplify as much as possible? (3 dimensions suffice here?). 

If you have have shape 4x6x8 then you can also check that the loop is on the dimension of size 4 (I think that's what your code is supposed to do).